### DISCUSSION TITLE
How to improve fast?

### DISCUSSION THREAD

**Discussion by shahtaha** [codewars/codewars.com#3176] [General]
I started learning JavaScript. What can i do to improve my skill in it And  i want to became a full stack developer... Can anybody guide me in this case?

**Comment by hobovsky** [comment]
Codewars Github is not meant for help with general programming or learning questions. It's meant to discuss matters related to Codewars platform, the website, its functions, etc.

For help with your questions please visit your favorite general programming forum, a general Javascript forum, or you can ask it in `#posts` or `#beginner` sections of Codewars Discord.

Closing as off-topic.
